.github/workflows/pg-s3-backup.yaml

```diff
@@ -0,0 +1,20 @@
+name: Publish pg-s3-backup
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "docker/pg-s3-backup/**"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Publish pg-s3-backup
+        uses: elgohr/Publish-Docker-Github-Action@2.18
```

Comment:
Why use this over the `docker/` actions?

Reply:
Evidently the official docker GH action was in the wasteland known as the second page of Google. Fixed now.